src/test_lib_json/main.cpp

```diff
@@ -2751,13 +2892,6 @@ JSONTEST_FIXTURE_LOCAL(ReaderTest, parseChineseWithOneError) {
   JSONTEST_ASSERT(reader.getFormattedErrorMessages() ==
                   "* Line 1, Column 19\n  Syntax error: value, object or array "
                   "expected.\n");
-  std::vector<Json::Reader::StructuredError> errors =
```

Comment:
why are these structured error verifications being deleted?

Reply:
I think getStructuredErrors() is not necessary for every testcase. Removing structured error verifications makes the testcase more clear and is convenient to add new testcase. In line 2819, we keep one structured error verifications for offset test.

Reply:
let's keep structured error checking and remove formatted error checking, which has less information anyway.
The structured errors can be checked by a helper function so we can make the tests compact.